awesomemotive/affiliatewp-store-credit
# Issue/135 - Bump WordPress minimum requirements

Issue #135


**Housekeeping**


**Proposed changes**


## How to test it
1. Install and activate Store Credit add-on
2. Confirm the plugin works as expected without any PHP errors.
3. Confirm existence and test saving the new settings on `Affiliates > Settings > Store Credit`
